imports/market-price-updates.js

```diff
@@ -32,6 +32,13 @@ async function sendUpdate(c) {
     msg += ` - B ${currency.format(btcUSDPrice.lastPrice, {code: 'USD'})}`;
   
     console.log(msg);
+
+    const lastMessage = retrieveLastMessage();
+
+    if (lastMessage.author.bot) {

```

Comment:
https://discord.js.org/#/docs/main/stable/class/User
`.author` points to a user object, which has the `bot` boolean.